index.html

```diff
@@ -0,0 +1,177 @@
+        <img
+          src="/images/logo.webp"
+          alt="판다마켓 로고"
+          width="153"
+          height="51"
+        />
+      </a>
+      <a href="/login.html" class="button login">로그인</a>
+    </header>
+    <main>
+      <section class="main-bg-section-wrapper">
+        <div class="layout main-bg-contents-wrapper">
+          <div class="main-bg-contents-container">
+            <p class="main-bg-contents-description">
+              일상의 모든 물건을<br />거래해 보세요
+            </p>
+            <a href="/items.html" class="button visit">구경하러 가기</a>
+          </div>
+          <img
+            src="/images/header-img.webp"
+            alt="판다마켓 메인 헤더"
+            width="746"
+            height="340"
+          />
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <img
+            src="/images/contents-img-01.webp"
+            alt="판다마켓 메인 컨텐츠 핫이슈"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Hot item</p>
+            <strong class="main-contents-strong"
+              >인기 상품을<br />확인해 보세요</strong
+            >
+            <p class="main-contents-description">
+              가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <div class="main-contents-container left">
+            <p class="main-contents-title">Search</p>
+            <strong class="main-contents-strong"
+              >구매를 원하는 <br />상품을 검색하세요</strong
+            >
+            <p class="main-contents-description">
+              구매하고 싶은 물품은 검색해서 <br />쉽게 찾아보세요
+            </p>
+          </div>
+          <img
+            src="/images/contents-img-02.webp"
+            alt="판다마켓 메인 컨텐츠 서치"
+            width="579"
+            height="444"
+          />
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <img
+            src="/images/contents-img-03.webp"
+            alt="판다마켓 메인 컨텐츠 레지스터"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Register</p>
+            <strong class="main-contents-strong"
+              >판매를 원하는 <br />상품을 등록하세요</strong
+            >
+            <p class="main-contents-description">
+              어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main-bg-section-wrapper">
+        <div class="layout main-bg-contents-wrapper">
+          <div class="main-bg-contents-container">
+            <p class="main-bg-contents-description">
+              믿을 수 있는 <br />판다마켓 중고 거래
```

Comment:
💊 제안

지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
              믿을 수 있는<br />판다마켓 중고 거래
```